README.md

```diff
@@ -1,7 +1,11 @@
 # Database Stream Processor
 
-Database Stream Processor (DBSP) is a framework for computing over data streams
-that aims to be more expressive and performant than existing streaming engines.
+Database Stream Processor (DBSP) is a framework for computing over *changing data*.
+That is, the computations must continuously update their outputs as new inputs arrive, in real time.
```

Comment:
computations that continuously
real-time is not hard real time. we should be clear about that